newjersey/navigator.business.nj.gov
# feat: [#188470579] Enable cemetery license status check

## Description
Enabling cemetery industry users license status search for the cemetery-cemetery license.



### Ticket



This pull request resolves [#188470579](https://www.pivotaltracker.com/story/show/188470579).

### Approach



### Steps to Test



- Create new cemetery business
- After business type, go to Apply for Your Certificate of Authority. You should see `Check Status` (WireMock has been updated with cemetery license for local use)

### Notes



- WireMock has been updated
- Spreadsheet has been updated

## Code author checklist
